script/Arcueid/Day 4/04_00_ARC04_6_2D.txt

```diff
@@ -304,8 +305,8 @@ I'm glad she did. She'd been staring at me with visible disgust in her eyes.
 [sha:73e00ae6f46c6f4e11ed01ef140d1d343b89ecb4]{
 -- Page 586, Offset 7141.
 -- 「あなたが見ているものはね、万物の結果、物の死に<易|やす>い箇所なのよ。もっと解りやすく言うのなら、あらゆる存在の死期……“死”そのものよ」
-"What you're seeing is the inevitable end of all things, the points where things can easily die.
- Put simply, you can see the fated end of anything that exists... 'death' itself."
+"What you're seeing is the finality of all things, the points where things can easily die.
```

Comment:
I wanted to avoid using 'end' too much. The ideas here presented with more nuance than that. It's trying to bring together a bunch of concepts related to 'death', and 結果 is one of them.